#!/usr/bin/env python
PACKAGE = "my_serial"

from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()

gen.add("target", int_t, 0, "target", 0, 0, 1000)
gen.add("lm_kp", double_t, 0, "kp", 0.005, 0, 5)
gen.add("lm_ki", double_t, 0, "ki", 0.0026, 0, 5)
gen.add("lm_kd", double_t, 0, "kd", 0.0, 0, 5)
gen.add("rm_kp", double_t, 0, "kp", 0.005, 0, 5)
gen.add("rm_ki", double_t, 0, "ki", 0.005, 0, 5)
gen.add("rm_kd", double_t, 0, "kd", 0.0, 0, 5)



exit(gen.generate(PACKAGE, "my_serial", "pid_params"))
